source/_integrations/incharge.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Vattenfall InCharge
+description: Instructions on how to integrate your Vattenfall InCharge managed charging stations within Home Assistant.
+ha_category:
+  - Energy
+  - Sensor
+ha_release: 2023.3.4
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@tgriek'
+ha_domain: incharge
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_integration_type: integration

```

Comment:
I think this is a service?

```suggestion
ha_integration_type: service

```

Reply:
Not sure? @klaasnicolaas ?

Reply:
No need to ping Klaas for that, see: <https://developers.home-assistant.io/blog/2022/10/24/hubs-devices-services-virtual-integrations>

Reply:
Klaas added a new-integration label that confused me. In any case I am fine with whatever suits the standards guys.

Reply:
> Klaas added a new-integration label that confused me.

It is new integration 🤷 
This defines the type of integration it is.

> whatever suits the standards guys.

It is not about standards.

Please note: What Klaas added are GitHub labels and have nothing to do with the contents/code itself.

Reply:
Gotcha. The idea and functioning behind this new integration is very similar to:
[Wallbox](https://www.home-assistant.io/integrations/wallbox/)
[Growatt](https://www.home-assistant.io/integrations/growatt_server/)
What makes those of the type integration and this one a service?

Reply:
Those are existing integrations that haven't specified them yet (please note how the linked blog post is just a couple of months old). For new integrations, it should just be set correctly from the get-go.